{"name": "Проверка раздела «Личный кабинет »", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x0044C1C3+27395]\n\t(No symbol) [0x003E3DC4]\n\t(No symbol) [0x002E1B7F]\n\t(No symbol) [0x00322C65]\n\t(No symbol) [0x00322D3B]\n\t(No symbol) [0x0035EC82]\n\t(No symbol) [0x003439E4]\n\t(No symbol) [0x0035CB24]\n\t(No symbol) [0x00343736]\n\t(No symbol) [0x00317541]\n\t(No symbol) [0x003180BD]\n\tGetHandleVerifier [0x00703A93+2876371]\n\tGetHandleVerifier [0x00757F5D+3221661]\n\tGetHandleVerifier [0x004CD634+556916]\n\tGetHandleVerifier [0x004D474C+585868]\n\t(No symbol) [0x003ECE04]\n\t(No symbol) [0x003E9818]\n\t(No symbol) [0x003E99B7]\n\t(No symbol) [0x003DBF0E]\n\tBaseThreadInitThunk [0x7570FCC9+25]\n\tRtlGetAppContainerNamedObjectPath [0x770680CE+286]\n\tRtlGetAppContainerNamedObjectPath [0x7706809E+238]", "trace": "tests\\test_profile_page.py:27: in test_go_to_order_history_in_personal_account\n    profile_page.click_on_order_history_button()\n        driver     = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"88813063965d22380dc049ca57fe43c9\")>\n        login      = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"88813063965d22380dc049ca57fe43c9\")>\n        main_page  = <pages.main_page.MainPage object at 0x00000200204767B0>\n        profile_page = <pages.personal_account_page.PersonalAccountPage object at 0x0000020020474770>\n        self       = <tests.test_profile_page.TestPersonalAccount object at 0x000002002032C2F0>\npages\\personal_account_page.py:12: in click_on_order_history_button\n    self.click_on_element(ProfilePageLocators.ORDER_HISTORY_BUTTON)\n        self       = <pages.personal_account_page.PersonalAccountPage object at 0x0000020020474770>\npages\\base_page.py:16: in click_on_element\n    WebDriverWait(self.driver, 50).until(EC.element_to_be_clickable(locator))\n        locator    = ('link text', 'История заказов')\n        self       = <pages.personal_account_page.PersonalAccountPage object at 0x0000020020474770>\nC:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: in until\n    raise TimeoutException(message, screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message: \nE   Stacktrace:\nE   \tGetHandleVerifier [0x0044C1C3+27395]\nE   \t(No symbol) [0x003E3DC4]\nE   \t(No symbol) [0x002E1B7F]\nE   \t(No symbol) [0x00322C65]\nE   \t(No symbol) [0x00322D3B]\nE   \t(No symbol) [0x0035EC82]\nE   \t(No symbol) [0x003439E4]\nE   \t(No symbol) [0x0035CB24]\nE   \t(No symbol) [0x00343736]\nE   \t(No symbol) [0x00317541]\nE   \t(No symbol) [0x003180BD]\nE   \tGetHandleVerifier [0x00703A93+2876371]\nE   \tGetHandleVerifier [0x00757F5D+3221661]\nE   \tGetHandleVerifier [0x004CD634+556916]\nE   \tGetHandleVerifier [0x004D474C+585868]\nE   \t(No symbol) [0x003ECE04]\nE   \t(No symbol) [0x003E9818]\nE   \t(No symbol) [0x003E99B7]\nE   \t(No symbol) [0x003DBF0E]\nE   \tBaseThreadInitThunk [0x7570FCC9+25]\nE   \tRtlGetAppContainerNamedObjectPath [0x770680CE+286]\nE   \tRtlGetAppContainerNamedObjectPath [0x7706809E+238]\n        end_time   = 462450.234\n        message    = ''\n        method     = <function element_to_be_clickable.<locals>._predicate at 0x000002002045A480>\n        screen     = None\n        self       = <selenium.webdriver.support.wait.WebDriverWait (session=\"88813063965d22380dc049ca57fe43c9\")>\n        stacktrace = ['\\tGetHandleVerifier [0x0044C1C3+27395]',\n '\\t(No symbol) [0x003E3DC4]',\n '\\t(No symbol) [0x002E1B7F]',\n '\\t(No symbol) [0x00322C65]',\n '\\t(No symbol) [0x00322D3B]',\n '\\t(No symbol) [0x0035EC82]',\n '\\t(No symbol) [0x003439E4]',\n '\\t(No symbol) [0x0035CB24]',\n '\\t(No symbol) [0x00343736]',\n '\\t(No symbol) [0x00317541]',\n '\\t(No symbol) [0x003180BD]',\n '\\tGetHandleVerifier [0x00703A93+2876371]',\n '\\tGetHandleVerifier [0x00757F5D+3221661]',\n '\\tGetHandleVerifier [0x004CD634+556916]',\n '\\tGetHandleVerifier [0x004D474C+585868]',\n '\\t(No symbol) [0x003ECE04]',\n '\\t(No symbol) [0x003E9818]',\n '\\t(No symbol) [0x003E99B7]',\n '\\t(No symbol) [0x003DBF0E]',\n '\\tBaseThreadInitThunk [0x7570FCC9+25]',\n '\\tRtlGetAppContainerNamedObjectPath [0x770680CE+286]',\n '\\tRtlGetAppContainerNamedObjectPath [0x7706809E+238]',\n '']"}, "description": "Проверка перехода в раздел «История заказов»", "steps": [{"name": "Нажатие на кнопку \"Личный кабинет\"", "status": "passed", "start": 1719579327821, "stop": 1719579328140}, {"name": "Нажимаем на кнопку \"История заказов\"", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x0044C1C3+27395]\n\t(No symbol) [0x003E3DC4]\n\t(No symbol) [0x002E1B7F]\n\t(No symbol) [0x00322C65]\n\t(No symbol) [0x00322D3B]\n\t(No symbol) [0x0035EC82]\n\t(No symbol) [0x003439E4]\n\t(No symbol) [0x0035CB24]\n\t(No symbol) [0x00343736]\n\t(No symbol) [0x00317541]\n\t(No symbol) [0x003180BD]\n\tGetHandleVerifier [0x00703A93+2876371]\n\tGetHandleVerifier [0x00757F5D+3221661]\n\tGetHandleVerifier [0x004CD634+556916]\n\tGetHandleVerifier [0x004D474C+585868]\n\t(No symbol) [0x003ECE04]\n\t(No symbol) [0x003E9818]\n\t(No symbol) [0x003E99B7]\n\t(No symbol) [0x003DBF0E]\n\tBaseThreadInitThunk [0x7570FCC9+25]\n\tRtlGetAppContainerNamedObjectPath [0x770680CE+286]\n\tRtlGetAppContainerNamedObjectPath [0x7706809E+238]\n\n", "trace": "  File \"C:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\ilme5\\OneDrive\\Рабочий стол\\учеба\\автоматизация\\Diplom\\Diplom_3\\pages\\personal_account_page.py\", line 12, in click_on_order_history_button\n    self.click_on_element(ProfilePageLocators.ORDER_HISTORY_BUTTON)\n  File \"C:\\Users\\ilme5\\OneDrive\\Рабочий стол\\учеба\\автоматизация\\Diplom\\Diplom_3\\pages\\base_page.py\", line 16, in click_on_element\n    WebDriverWait(self.driver, 50).until(EC.element_to_be_clickable(locator))\n  File \"C:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1719579328140, "stop": 1719579378427}], "attachments": [{"name": "log", "source": "ad5719a1-104a-413f-8cc5-764042978a8c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "driver", "value": "'chrome'"}], "start": 1719579327821, "stop": 1719579378430, "uuid": "86af9365-4d5b-43d2-a63f-d56c627281df", "historyId": "8b49eabcd0e5949d8374fce5a54568d7", "testCaseId": "f51e20e9f238aeb41ffc5bb5847e4767", "fullName": "tests.test_profile_page.TestPersonalAccount#test_go_to_order_history_in_personal_account", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_profile_page"}, {"name": "subSuite", "value": "TestPersonalAccount"}, {"name": "host", "value": "DESKTOP-8VJF26I"}, {"name": "thread", "value": "11160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_profile_page"}]}